src/workloads/execution/BackgroundTTLDeletions.yml

```diff
@@ -0,0 +1,187 @@
+          Duration: 3 minutes
+          CollectionCount: *NumColls
+          Operations:
+          - OperationName: insertMany
+            OperationCommand:
+              Documents:  # Insert 50 documents at a time
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+              - *generatedDoc
+
+
+Actors:
+- Name: Setup
+  Type: AdminCommand
+  Threads: 1
+  Phases:
+    OnlyActiveInPhases:
+      Active: [1]
+      NopInPhasesUpTo: *NumPhases
+      PhaseConfig:
+        Repeat: 1
+        Database: admin
+        Operations:
+        - OperationName: AdminCommand
+          OperationCommand:
+            setParameter: 1
+            ttlMonitorSleepSecs: 1
+        - OperationName: AdminCommand
+          OperationCommand:
+            setParameter: 1
+            storageEngineConcurrentWriteTransactions: 32
+        SleepAfter: 60 seconds  # Wait 60 seconds so that the new ttl sleep config is active.
+
+- Name: InsertData
+  Type: Loader
+  Threads: 1
+  Phases:
+    OnlyActiveInPhases:
+      Active: [2]
+      NopInPhasesUpTo: *NumPhases
+      PhaseConfig:
+        Repeat: 1
+        Database: *db
+        Threads: 1
+        CollectionCount: *NumColls
+        DocumentCount: 50000
+        BatchSize: 10000
+        Document: *generatedDoc
+        Indexes:
+        - keys: {expireDate: 1}
+          options: {expireAfterSeconds: 0}
+
+- ActorFromTemplate:
+    TemplateName: WritesActorTemplate
+    TemplateParameters:
+      Name: Writes_1_Thread
+      Threads: 1
+      ActivePhases: [3]
```

Comment:
I assume we actively do not want to quiesce between phases here, because that would allow the ttl process to catch up and keep working. Is that correct? If it is, it's worth adding a comment somewhere in this file to that effect.

Reply:
You're right in that we want the TTL deleter to be working constantly for this test. I've added a comment explaining this behaviour.